vita3k/gui/src/app_context_menu.cpp

```diff
@@ -352,11 +352,11 @@ void draw_app_context_menu(GuiState &gui, EmuEnvState &emuenv, const std::string
             if (!emuenv.cfg.show_live_area_screen && ImGui::BeginMenu("Live Area")) {
                 if (ImGui::MenuItem("Live Area", nullptr, &gui.live_area.live_area_screen))
                     open_live_area(gui, emuenv, app_path);
-                if (ImGui::MenuItem("Search", nullptr))
+                if (ImGui::MenuItem(lang["search"].c_str(), nullptr))
                     open_search(APP_INDEX->title);
-                if (fs::exists(MANUAL_PATH) && !fs::is_empty(MANUAL_PATH) && ImGui::MenuItem("Manual", nullptr))
+                if (fs::exists(MANUAL_PATH) && !fs::is_empty(MANUAL_PATH) && ImGui::MenuItem(lang["manual"].c_str(), nullptr))
                     open_manual(gui, emuenv, app_path);
-                if (ImGui::MenuItem("Update"))
+                if (ImGui::MenuItem(lang["update"].c_str()))
```

Comment:
no need change this, is just replic of button

Reply:
What do you mean?
This place is essentially an icon, right?
If you want to make it an icon, you should, and if not, why not translate it?

Reply:
Because is icon without any name on psvita, and no can translate it in other language.
Is possible inside App contexte, but is only duplicate for live area, so no have any utility

Reply:
[https://manuals.playstation.net/document/en/psvita/basic/screens.html](url)
This page is available in various languages, so an official translation is possible, I think.

Reply:
Ah i see i have no know that lol, but is not possible add translate for button in live area, that relate size of Word, but can add it for app context, but is just réplicate of live area button so no have réal utility 🤷‍♂️

Reply:
OK, i got it.